test/autorest/generated/additionalpropertiesgroup/internal/additionalpropertiesgroup/models.go

```diff
@@ -42,79 +43,84 @@ func (e Error) Error() string {
 }
 
 type PetApInProperties struct {
-	// Dictionary of <components·schemas·petapinproperties·properties·additionalproperties·additionalproperties>
+	// Dictionary of <number>
 	AdditionalProperties *map[string]*float32 `json:"additionalProperties,omitempty"`
-	ID *int32 `json:"id,omitempty"`
-	Name *string `json:"name,omitempty"`
-	Status *bool `json:"status,omitempty"`
+	ID                   *int32               `json:"id,omitempty"`
+	Name                 *string              `json:"name,omitempty"`
+	Status               *bool                `json:"status,omitempty"`
 }
 
 type PetApInPropertiesWithApstring struct {
-	// Dictionary of <components·schemas·petapinproperties·properties·additionalproperties·additionalproperties>
+	// Dictionary of <number>
 	AdditionalProperties *map[string]*float32 `json:"additionalProperties,omitempty"`
-	ID *int32 `json:"id,omitempty"`
-	Name *string `json:"name,omitempty"`
-	OdataLocation *string `json:"@odata.location,omitempty"`
-	Status *bool `json:"status,omitempty"`
+	ID                   *int32               `json:"id,omitempty"`
+	Name                 *string              `json:"name,omitempty"`
+	OdataLocation        *string              `json:"@odata.location,omitempty"`
+	Status               *bool                `json:"status,omitempty"`
 }
 
 type PetApObject struct {
-	ID *int32 `json:"id,omitempty"`
-	Name *string `json:"name,omitempty"`
-	Status *bool `json:"status,omitempty"`
+	ID     *int32  `json:"id,omitempty"`
+	Name   *string `json:"name,omitempty"`
+	Status *bool   `json:"status,omitempty"`
 }
 
 type PetApString struct {
-	ID *int32 `json:"id,omitempty"`
-	Name *string `json:"name,omitempty"`
-	Status *bool `json:"status,omitempty"`
+	ID     *int32  `json:"id,omitempty"`
+	Name   *string `json:"name,omitempty"`
+	Status *bool   `json:"status,omitempty"`
 }
 
 type PetApTrue struct {
-	ID *int32 `json:"id,omitempty"`
-	Name *string `json:"name,omitempty"`
-	Status *bool `json:"status,omitempty"`
+	ID     *int32  `json:"id,omitempty"`
+	Name   *string `json:"name,omitempty"`
+	Status *bool   `json:"status,omitempty"`
 }
 
 // PetsCreateApInPropertiesResponse contains the response from method Pets.CreateApInProperties.
 type PetsCreateApInPropertiesResponse struct {
 	PetApInProperties *PetApInProperties
```

Comment:
Should this have comment over it?

Reply:
So there is a description defined in the swagger.
```json
          "200": {
            "description": "A Pet which contains more properties than what is defined.",
            "schema": {
              "$ref": "#/definitions/PetAPInProperties"
            }
          },
```
However I don't see this in the model.  I'll follow up about that.